kitsune/questions/views.py

```diff
@@ -533,8 +534,20 @@ def aaq(request, product_key=None, category_key=None, step=1):
 
             return HttpResponseRedirect(path)
 
-        if has_subscriptions:
-            zendesk_form = ZendeskForm(data=request.POST or None, product=product)
+        # If product allows loginless support
+        #  and not logged in, use CATEGORY_CHOICES_LOGINLESS
+        if product_key in settings.LOGINLESS_PRODUCTS and not request.user.is_authenticated:
```

Comment:
Since this check refers to the `Zendesk` form, maybe it should come after the next check where we try to figure out if we need support through zendesk or the forums

Reply:
The check for the existence of the `product_key` in the list happens in more than one places. Shall we replace it with something like
`render_loginless_categories = product_key in settings.LOGINLESS_PRODUCTS`

Reply:
Added `is_loginless`